src/util/cmdline.cpp

```diff
@@ -195,12 +234,33 @@ bool cmdlinet::parse(
   p.add("input-file", -1);
   try
   {
+    // Load env
     boost::program_options::store(
       boost::program_options::command_line_parser(
         simple_shell_unescape(getenv("ESBMC_OPTS"), "ESBMC_OPTS"))
         .options(all_cmdline_options)
         .run(),
       vm);
+    // Load config file
+    const auto config_path = this->get_config_file_location();
+    if (!config_path.empty())
+    {
+      std::ifstream file(config_path);
+      if (file)
+      {
+        log_status("Reading config file: {}", config_path);
+      }
+      else
+      {
+        log_error("Could not read config file: {}", config_path);
```

Comment:
let's not fail if there is no config file.

Reply:
This fails only if the location of the environment variable is set, in which case, it shows that the user had a clear intention to load the config file. If it is unset, it will not fail. Should I change it from this behaviour?

Reply:
I have fixed the bug now!